credentialMixin.js

```diff
@@ -9,6 +9,10 @@ export const credentialMixin = {
       type: Object,
       required: true,
     },
+    profile: {
```

Comment:
`profile` should be removed as a prop. We just need the color here, right?

Reply:
We need profile in order to show the name of the profile in the credentials details.